wait/exit.go

```diff
@@ -46,8 +46,9 @@ func (ws *ExitStrategy) WithPollInterval(pollInterval time.Duration) *ExitStrate
 //
 // For Example:
 // wait.
-//     ForExit().
-//     WithPollInterval(1 * time.Second)
+//
```

Comment:
Is this empty line required by golangci-lint?

Reply:
Good catch! This was applied because `gofmt` interpreted `wait.` as the end of a paragraph block, with the indented block as the code block. I've resolved this via https://github.com/testcontainers/testcontainers-go/commit/a6b8efe43540cd894b21521fbf16a8bd122de585

Reply:
resolved via [4733139](https://github.com/testcontainers/testcontainers-go/pull/569/commits/47331393f3cccb144105a759471f256a8ff820cf)